collector/textfile.go

```diff
@@ -34,6 +35,7 @@ import (
 
 var (
 	textFileDirectory = kingpin.Flag("collector.textfile.directory", "Directory to read text files with metrics from.").Default("").String()
+	addOnce           sync.Once
```

Comment:
Nit: with this being at the package level, `addOnce` is pretty generic. How about `textfileAddOnce`?

Reply:
looks reasonable, i will fix and resubmit.